examples/standalone/cost.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, problem=None):
         )
         self.x0 = self.parameters.initial_value()
 
-    def _evaluate(self, inputs):
+    def compute(self, inputs):
```

Comment:
It would be great to keep similar names for `evaluate` and `_evaluate` for now, until we have worked on #436. Perhaps `evaluate_on_prediction` or `evaluate_with_data`?